processing/data_collection/gazette/spiders/df_brasilia.py

```diff
@@ -0,0 +1,85 @@
+import re
+import json
+import itertools
+import dateparser
+
+from datetime import datetime
+from scrapy import Request
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class DfBrasiliaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "5300108"
+    GAZETTE_URL = "http://dodf.df.gov.br/listar"
+
+    MONTHS = {
+        "12": "12_Dezembro",
+        "11": "11_Novembro",
+        "10": "10_Outubro",
+        "09": "09_Setembro",
+        "08": "08_Agosto",
+        "07": "07_Julho",
+        "06": "06_Junho",
+        "05": "05_Maio",
+        "04": "04_Abril",
+        "03": "03_Março",
+        "02": "02_Fevereiro",
+        "01": "01_Janeiro",
+    }
+    YEARS = ["2018", "2017", "2016", "2015"]
+    MONTHS_YEARS = itertools.product(MONTHS.values(), YEARS)
```

Comment:
You also will have problems hard-coding the months. For example, at the moment of this review, if you try to get the data for december/2018 you will get a 404 error.